src/bin/pg_dump/dump_babel_utils.c

```diff
@@ -1124,9 +1124,9 @@ addFromClauseForLogicalDatabaseDump(PQExpBuffer buf, TableInfo *tbinfo)
 						  "ON a.database_name = b.name COLLATE \"C\" "
 						  "WHERE b.dbid = %d "
 						  "AND a.rolname NOT IN "
-						  "('master_dbo', 'master_db_owner', 'master_guest', "
-						  "'msdb_dbo', 'msdb_db_owner', 'msdb_guest', "
-						  "'tempdb_dbo', 'tempdb_db_owner', 'tempdb_guest') ",
+						  "('master_dbo', 'master_db_owner', 'master_guest', 'master_db_accessadmin', "
+						  "'msdb_dbo', 'msdb_db_owner', 'msdb_guest', 'msdb_db_accessadmin', "
+						  "'tempdb_dbo', 'tempdb_db_owner', 'tempdb_guest', 'tempdb_db_accessadmin') ",
```

Comment:
Would appreciate if you can refactor these default role to a static array like we have `default_bbf_roles` in dumpall_babel_utils.c

Reply:
Done